inception-project/inception
# #1338 - Support telemetry

**What's in the PR**
- Enable telemetry support

**How to test manually**
* Log in as admin, see if telemetry settings popup
* Make settings and save
* Access settings again via heartbeat icon in footer

**Automatic testing**

**Documentation**
